types/react-sizes/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
```

Comment:
I don't think this is necessary, is it?

Reply:
good question, will take a look.

Reply:
https://www.typescriptlang.org/docs/handbook/compiler-options.html

https://stackoverflow.com/questions/39303385/what-does-the-tsconfig-option-lib-do

Reply:
alright, removed the lib compilerOption.  let's see if the tests pass 🤞

Reply:
@andrewbranch looks like this `"lib": ["es6"]` compiler option is required after all.  

```
npm run test
# ...
=== ERRORS ===



Error in react-sizes
Error: Must specify "lib", usually to `"lib": ["es6"]` or `"lib": ["es6", "dom"]`.
    at Object.<anonymous> (/Users/m/workspace/DefinitelyTyped/node_modules/dtslint/bin/checks.js:102:19)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/m/workspace/DefinitelyTyped/node_modules/dtslint/bin/checks.js:4:58)
Error: The following packages had errors: react-sizes
    at doRunTests (/Users/m/workspace/DefinitelyTyped/node_modules/types-publisher/src/tester/test-runner.ts:235:11)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at Object.runTests [as default] (/Users/m/workspace/DefinitelyTyped/node_modules/types-publisher/src/tester/test-runner.ts:94:5)
    at main (/Users/m/workspace/DefinitelyTyped/node_modules/types-publisher/src/tester/test.ts:26:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! definitely-typed@0.0.3 test: `node node_modules/types-publisher/bin/tester/test.js --run-from-definitely-typed`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the definitely-typed@0.0.3 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/m/.npm/_logs/2019-07-03T02_18_21_892Z-debug.log
```

Reply:
no worries, thanks for the review ☺️